### DISCUSSION TITLE
Access control on /api/1/users

### DISCUSSION THREAD

**Discussion by gpsilv4** [opendatateam/udata#3033] [Q&A]
It has been verified that the application present at data.gouv.fr is not validating permissions correctly of users.

It is possible for any unauthenticated user to view all users.

Page where the access control failure was identified:
- /api/1/users/

Suggestion:
The application must validate the user's role/position and permissions and ensure that only those with the due permissions.

**Comment by maudetes** [comment]
Thank you for the notification! See the dedicated issue we've created: https://github.com/opendatateam/udata/issues/3044 (planned in this sprint)

**Comment by maudetes** [comment]
We've added the need to have admin permissions to acces the `api/1/users` endpoint in https://github.com/opendatateam/udata/pull/3047.

**Comment by gpsilv4** [comment]
I can still see the users without being logged into the portal.

![Capturar](https://github.com/opendatateam/udata/assets/31599682/22a30db2-192c-4ab9-a82e-3c5ae0efebea)

**Reply by maudetes** [reply]
It wasn't deployed in production already. It has been deployed since.
